app/models/supplier.rb

```diff
@@ -71,7 +71,7 @@ def update_articles_from_file(file, **opts)
         article.destroy && outlisted_counter += 1
 
       # remember special info for article; store data to allow article after its special
-      elsif status == :special
+      elsif status == :special and article
```

Comment:
for the boolean operator, please use `&&`

Reply:
I used the same operator as it was used in line [65](https://github.com/robwa/sharedlists/blob/7c35a7dd3f1b57aa87adb2a0f0b707e550993fed/app/models/supplier.rb#L65) and [70](https://github.com/robwa/sharedlists/blob/7c35a7dd3f1b57aa87adb2a0f0b707e550993fed/app/models/supplier.rb#L70) above. Change only this one?

Reply:
Wow, I missed that. Thanks for keeping to local style! :)
I'd say it's better to change all of them, but feel free to take your pick. Happy to merge when ready.

Reply:
I change the operators in a separate commit, to make clear that it's only refactoring.